rgd/geodata/templates/geodata/_includes/search_form.html

```diff
@@ -73,6 +74,16 @@
     </form>
   </div>
 
+  <div id="GeoJSON" class="tabcontent">
+    <form method="GET">
+      GeoJSON geometry: <input type="text" name="geojson" value="{{ request.GET.geojson }}">
+      <br>
+      within: <input type="checkbox" {{ request.GET.within|yesno:"checked,," }} name="within">
```

Comment:
Let's add a little tooltip

```suggestion
      within: <input type="checkbox" {{ request.GET.within|yesno:"checked,," }} name="within" title="Only retrieve entries that are fully within the boundary of the feature.">
```